### PR TITLE
Add bitcore examples to the README

### LINKED ISSUE #1015: Add more code examples to the README
The basic usecases must be covered. Take a look at [this examples](https://github.com/bitcoinjs/bitcoinjs-lib#examples).

### PR DESCRIPTION
**Pull request by yemel**
Closes #1015

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    .sign(privateKey);
+```
+
+## Create a 2-of-3 multisig P2SH address
+```javascript
+var publicKeys = [
+  '026477115981fe981a6918a6297d9803c4dc04f328f22041bedff886bbc2962e01',
+  '02c96db2302d19b43d4c69368babace7854cc84eb9e061cde51cfa77ca4a22b8b9',
+  '03c6103b3b83e4a24a0e33a4df246ef11772f9992663db0c35759a5e2ebf68d8e9'
+].map(bitcore.PublicKey);
```
Comment (eordano) [inline docs/examples.md]: The `map` is not necessary

File: docs/examples.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+```
+
+## Spend from a 2-of-2 multisig P2SH address
+```javascript
+var privateKeys = [
+  new bitcore.PrivateKey('91avARGdfge8E4tZfYLoxeJ5sGBdNJQH4kvjJoQFacbgwmaKkrx'),
+  new bitcore.PrivateKey('91avARGdfge8E4tZfYLoxeJ5sGBdNJQH4kvjJoQFacbgww7vXtT')
+];
+var publicKeys = privateKeys.map(bitcore.PublicKey);
+var address = new bitcore.Address(publicKeys, 2); // 2 of 2
```
Comment (eordano) [inline docs/examples.md]: Not used?

**Comment by eordano** [comment]
Looks great

File: docs/examples.md (hunk 2 above)
Comment (yemel) [inline docs/examples.md]: updated

**Comment by yemel** [comment]
updated

**Pull request merged**